pyproject.toml

```
@@ -13,8 +13,8 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.poetry.dependencies]
 python = "^3.12"
-django = "~5.0.6"
-wagtail = "~6.2.2"
+django = "~5.0.9"
```

why not 

```suggestion
django = "~5.1.2"
```

as well?

---

@zerolab that is blocked waiting for a new release of https://github.com/torchbox/django-pattern-library

---

argh, I keep forgetting about it.
one workaround is to build it locally and bundle the wheel with the repo, but that is not worth it. One for a follow up PR